recipes/apache-solr/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "9.7.0" %}
+package:
+  name: apache-solr
+  version: {{ version }}  # Use the specific version you want to package
+
+source:
+  - url: https://downloads.apache.org/solr/solr/{{ version }}/solr-{{ version }}.tgz
+    sha256: 38548b86fa4e3c87883875952da124bf7d742cb8f7b25d37a1176833588e8552
+  - url: https://raw.githubusercontent.com/apache/solr/refs/heads/main/LICENSE.txt
+    sha256: 7feb4036d88cfa5fae13a0d55e7638a700ef77d2b4e6fcebaacac889081a7923
+  - url: https://raw.githubusercontent.com/apache/solr/refs/heads/main/NOTICE.txt
+    sha256: 15b37dbc8a5b68547c4a9d93fc1e8a72609e2d31d59e5d6f57241cee988e073a
+build:
+  number: 0
+  noarch: generic
```

Comment:
You need noarch_variants (see the docs) for Windows and Unix. The generated package isn't the same.

Reply:
I've tried to run the build stage only for linux, as that should be enough. But it seems I don't understand the skip directive.